### PR TITLE
TECH-559: Add Tests For GET /services/serviceId

### PR DESCRIPTION
**Pull request by mdulko-soldevelo**
Ticket: https://govstack-global.atlassian.net/browse/TECH-559

### CODE REVIEW

File: examples/mock/mockoon-registration.json [hunk 1]
```diff
@@ -160,11 +160,11 @@
       "type": "http",
       "documentation": "Pass in the ID of the service and it will return all information about that service",
       "method": "get",
-      "endpoint": "services/{serviceKey}",
+      "endpoint": "services/{serviceId}",
```
Comment (KarolinaKopacz) [inline examples/mock/mockoon-registration.json]:
```suggestion
      "endpoint": "services/:serviceId",
```

File: test/openAPI/features/support/helpers/service_serviceId.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  () =>
+    specServiceId
+      .response()
+      .to.have.responseTimeLessThan(defaultExpectedResponseTime)
+);
+
+Then(/^The \/services\/\{serviceId\} response should have status (\d+)$/, (status) => {
+ console.log(specServiceId)
+  specServiceId.response().to.have.status(status)
+});
```
Comment (KarolinaKopacz) [inline test/openAPI/features/support/helpers/service_serviceId.js]:
```suggestion
Then(/^The \/services\/\{serviceId\} response should have status (\d+)$/, (status) => 
  specServiceId.response().to.have.status(status)
);
```

File: test/openAPI/features/support/helpers/helpers.js [hunk 3]
```diff
@@ -78,4 +78,5 @@ module.exports = {
     required: ['id', 'name', 'type', 'version'],
     additionalProperties: false,
   },
+  serviceIdEndpoint: 'services/{serviceId}'
```
Comment (KarolinaKopacz) [inline test/openAPI/features/support/helpers/helpers.js]: Please add above comment `// service_serviceId`
and also we can moved `serviceSchema` to the end of the file  and add comment `// shares`

File: test/openAPI/features/services_serviceId.feature [hunk 4]
```diff
@@ -0,0 +1,12 @@
+@method=GET @endpoint=/services/{serviceId}
+Feature: API endpoint that returns an information of single service
+
+  @smoke @unit @positive
+  Scenario: Retrieve the information of single service smoke type test
+
+    Given User wants to retrieve the service
```
Comment (KarolinaKopacz) [inline test/openAPI/features/services_serviceId.feature]:
```suggestion
    Given User wants to retrieve all the information about the service
```

**Review by KarolinaKopacz — CHANGES_REQUESTED** [review]

**Review by KarolinaKopacz — APPROVED** [review]

**Comment by dbienkowska** [comment]
@ingmarvali could you please review this PR?

**Review by ingmarvali — APPROVED** [review]

**Pull request merged**